assembly/frame/system.ts

```diff
@@ -135,6 +158,9 @@ export class System {
         SystemStorageEntries.Digest().set(new ByteArray(digests));
         const blockNumber: BlockNumber = instantiate<BlockNumber>((<BlockNumber>header.getNumber()).unwrap() - 1);
         SystemStorageEntries.BlockHash().set(header.getParentHash(), blockNumber);
+        
+        // Kill inspectable storage entries in state
+        Storage.clear(Utils.getHashedKey("System", "Events", null));
```

Comment:
can we use the `Events()` method? My thinking is that we should have at best only one place in which we hardcode te `System` `Events` key